readthedocs/api/v3/views.py

```diff
@@ -244,11 +248,40 @@ class SubprojectRelationshipViewSet(APIv3Settings, NestedViewSetMixin,
     ``parents_query_lookups`` defined when registering the urls.
     """  # noqa
 
-    model = Project
-    lookup_field = 'slug'
-    lookup_url_kwarg = 'project_slug'
-    serializer_class = ProjectSerializer
-    queryset = Project.objects.all()
+    model = ProjectRelationship
+    lookup_field = 'alias'
+    lookup_url_kwarg = 'alias_slug'
+    queryset = ProjectRelationship.objects.all()
+
+    def get_serializer_class(self):
+        """
+        Return correct serializer depending on the action.
+
+        For GET it returns a serializer with many fields and on POST,
+        it return a serializer to validate just a few fields.
+        """
+        if self.action == 'create':
```

Comment:
aren't these constants somewhere?

Reply:
I've never seen them used as constants. I did a quick grep on their source code and I found that they use just the strings as well :'( : https://github.com/encode/django-rest-framework/blob/e57c1505fc4ed957332ae547a35c8713acfdf30c/rest_framework/schemas/coreapi.py#L37